### ISSUE TITLE
Trt slower than pytorch

### ISSUE THREAD

**Issue by francesco-taioli** [NVIDIA-AI-IOT/torch2trt#695]
Hi,
the inference of  **trt  (0.70 sec)  is slower than pytorch (0.04 sec)**. The memory usage, instead, is slower by a large margin on trt.

After having enabled the profiler, inside the torch2trt module, I found this line that take a huge amount of time.

`[CONVOLUTION #2] torch.nn.Conv2d.forward(Conv2d(128, 128, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1), groups=128), tensor(shape=[1, 128, 64, 64], dtype=torch.float32)) + [ELEMENTWISE #29] torch.Tensor.__add__(tensor(shape=[1, 128, 64, 64], dtype=torch.float32), tensor(shape=[1, 128, 64, 64], dtype=torch.float32)): 601.418ms`

This is the [repo](https://github.com/francesco-taioli/CCTrans). Look at [torch_to_trt.py](https://github.com/francesco-taioli/CCTrans/blob/main/torch_to_trt.py)

There is something else I can do?
